docs/evaluation/how_to_guides/evaluation/use_prebuilt_llm_evaluator.mdx

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 14
+---
+
+# Use Prebuilt LLMEvaluator (Python only)
+
+:::tip Recommended Reading
+Before diving into this content, it might be helpful to read the following:
+
+- [LangChain evaluator reference](/reference/sdk_reference/langchain_evaluators)
+
+:::
+
+When working with applications that produce unstructured text outputs, it can be challenging to assess the quality and accuracy of those outputs without involving a human expert or another LLM.
+Many applications need to evaluate responses on common criteria such as correctness, hallucination detection, conciseness, and bias.
+To simplify and standardize this process, LangSmith provides the `LLMEvaluator` class, a versatile LLM-as-a-judge tool.
+With `LLMEvaluator`, you can create automated and customizable evaluations for your LLM outputs, giving you structured feedback on specific aspects of each response.
+
+:::tip Prerequisites
+
+Create a dataset and set up the LangSmith client in Python to follow along
```

Comment:
You can create a public dataset and let people clone it. I have a docs repo I made I can share

Reply:
oh yeah that would be nice